Block/Catalog/Product/Installments.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace RicardoMartins\PagSeguro\Block\Catalog\Product;
+
+use Magento\Framework\View\Element\Template;
+use Magento\Catalog\Helper\Data;
+
+class Installments extends Template
+{
+    protected $helper;
+
+    public function __construct(
+        Template\Context $context,
+        Data $helper,
+        array $data = [])
+    {
+        $this->helper = $helper;
+        parent::__construct($context, $data);
+    }
+    public function getInstallmentsText()
+    {
+        $objectManager = \Magento\Framework\App\ObjectManager::getInstance();
```

Comment:
Please don't use ObjectManager to create objects on the fly. It's against Magento standards and probably will be rejected when going through Magento's Marketplace quality tool. 
Inject the class/property in the __construct instead.